.gitignore

```diff
@@ -43,6 +43,6 @@ htmlcov/
 *.tfstate.*.backup
 .vscode
 .env
-venv/
-.venv/
+venv*/
```

Comment:
I wanted to support multiple local venvs with different versions of Python, for example `venv3.9.10` and `venv3.10.7`.